app/serializers/activitypub/flag_serializer.rb

```diff
@@ -20,7 +20,15 @@ def virtual_object
     [ActivityPub::TagManager.instance.uri_for(object.target_account)] + object.statuses.map { |s| ActivityPub::TagManager.instance.uri_for(s) }
   end
 
+  def summary
+    object.category
+  end
+
   def content
-    object.comment
+    if object.violation?
+      "Comment:\n#{object.comment}\n\nViolated #{Rails.configuration.x.local_domain}'s Rules: \n- #{object.rules.map(&:text).join("\n -")}"
```

Comment:
I18N considerations on this string?

Reply:
@mjankowski this will be superseded by https://github.com/swicg/activitypub-trust-and-safety/issues/2

Reply:
In otherwords, there won't be i18n here, since we won't be adding text to the comment of a report, instead we'll have an actual field through which to communicate the categorisation of a report.